app/src/main/java/de/tum/in/tumcampusapp/component/ui/news/TopNewsStore.kt

```diff
@@ -84,7 +91,7 @@ class RealTopNewsStore @Inject constructor(
         // --> Card should be displayed again
         val update = oldShowUntil != newsAlert.displayUntil || oldImage != newsAlert.url
         if (update) {
-            sharedPrefs.edit().putBoolean(CardManager.SHOW_TOP_NEWS, true).apply()
+            setEnabled(true)
         }
 
         sharedPrefs.edit().putString(Const.NEWS_ALERT_SHOW_UNTIL, newsAlert.displayUntil).apply()
```

Comment:
Why aren't we updating this value in the SharedPrefs when we are updating the image url and link a few lines before? Shouldn't make a difference and would look nicer